fluentlenium-core/src/test/java/org/fluentlenium/integration/FluentLeniumWaitTest.java

```diff
@@ -152,37 +182,37 @@ public void checkAwaitContainsTextWithTextMatcher() {
 
     @Test
     public void when_a_element_is_not_present_then_isNotPresent_return_true() {
-        await().atMost(1, NANOSECONDS).until(".small").withText().contains("notPresent").isNotPresent();
+        await().atMost(1, NANOSECONDS).until(".small").withText().contains("notPresent").not().isPresent();
```

Comment:
Problem is that this method name is in an interface, and it would seems strange in other use case to have present() only. Conditions are all prefixed with `is` or `has`, I think we should stick with this consistently.